addons/sms/static/src/components/notification_list/notification_list_notification_group_tests.js

```diff
@@ -107,16 +107,16 @@ QUnit.test('notifications grouped by notification_type', async function (assert)
     this.data['mail.message'].records.push(
         // first message that is expected to have a failure
         {
-            id: 11, // random unique id, will be used to link failure to message
-            message_type: 'email', // different type from second message
+            id: 12, // random unique id, will be used to link failure to message
+            message_type: 'sms', // different type from second message
```

Comment:
Isn't it useless diff in all of these test files?
It's inverting both the id and the type, leading to no change?

Reply:
You are right, I got tings mixed up here